week-3/E-array-map/exercise.js

```diff
@@ -3,3 +3,5 @@
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+console.log(numbers.map(x => x*100));
+
```

Comment:
The exercise asks you to write the solution multiple times using different syntax. Can you try using the `function() {}` syntax too?